### PR TITLE
Use fixed version of psalm

### PR DESCRIPTION
**Pull request by Nyholm**
3.14.2 was the first tagged version

### CODE REVIEW

File: psalm.baseline.xml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+<files psalm-version="3.14.2@3538fe1955d47f6ee926c0769d71af6db08aa488">
   <file src="src/Client.php">
-    <NullArgument occurrences="1">
-      <code>null</code>
-    </NullArgument>
     <PossiblyUndefinedVariable occurrences="1">
       <code>$options</code>
     </PossiblyUndefinedVariable>
+    <UndefinedConstant occurrences="1">
+      <code>\IDNA_DEFAULT</code>
```
Comment (GrahamCampbell) [inline psalm.baseline.xml:8]: The new docker image seems to mean we loose all static analysis of extensions not installed in the image... Not a problem, for undefined constants, but really unfortunately for all the function calls that are no longer analyzed...

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: psalm.baseline.xml (hunk 1 above)
Comment (Nyholm) [reply psalm.baseline.xml:8]: Which function calls?

Comment (GrahamCampbell) [reply psalm.baseline.xml:8]: All functions from extensions not installed in the image are no longer have their uses type-checked. Like ext-intl.

Comment (Nyholm) [reply psalm.baseline.xml:8]: I only find this: https://github.com/guzzle/guzzle/blob/879a6c664fda9d33f274d7a9b3cd65ac2f1b62a6/src/Utils.php#L389

Comment (GrahamCampbell) [reply psalm.baseline.xml:8]: Right, but I suppose my point was what other extensions are being skipped now, that we didn't notice?

**Comment by Nyholm** [comment]
You are correct. There is no win using this PR.

**Pull request closed without merging**